src/lib/educationalResources/educationalResources.ts

```diff
@@ -742,6 +742,40 @@ export const educationalResources: (
                 "deploymentUrl":
                     "https://datalab.sspcloud.fr/launcher/inseefrlab-helm-charts-trainings/neverending?autoLaunch=true&onyxia.friendlyName=«neveRending»",
             },
+             {
+                "name": {
+                    "fr": "Grimoire IgoR - Chapitre Bonus",
+                    "en": "IgoR Spellbook - Bonus Chapter",
+                    "it": "Grimorio IgoR - Capitolo bonus",
+                },
+                "abstract": {
+                    "fr": "Partie pédagogique du FuncampR. Chapitre Bonus :  le village de Phocea. La cheffe-Sylphe, Lireva, demande à icaRius de l'aider à reconstruire son village, détruit par les bombes sylvestres",
+                    "en": "FuncampR educational part. Bonus Chapter: the village of Phocea. The Sylph leader, Lireva, asks icaRius to help her rebuild her village, destroyed by Sylvan bombs",
+                    "it": "Parte educativa di FuncampR. Capitolo bonus: il villaggio di Phocea. La leader dei Silfi, Lireva, chiede a IcaRius di aiutarla a ricostruire il suo villaggio, distrutto dalle bombe silvane",
+                },
+                "authors": [
+                    {
+                        "fr": "Communauté FuncampR",
+                        "it": "Comunità FunCampR",
+                        "en": "FuncampR Community",
+                    },
+                ],
+                "types": [
+                    {
+                        "fr": "Tutoriel R",
+                        "it": "Tutorial R",
+                        "en": "R Tutorial",
+                    },
+                ],
+                "tags": ["learn"],
+                "category": "training courses with R",
+                "imageUrl": grimoire14ImgUrl,
```

Comment:
Il manque un import en début de fichier pour définir la variable grimoire14ImgUrl (cf les imports déjà présents)

Reply:
Bien noté (désolé pour l'oubli). Normalement un nouveau commit ajoute le lien d'import.